### ISSUE TITLE
Custom Url For Zeromint Collections

### ISSUE THREAD

**Issue by wwitkdqsck** [zeromintapp/community#7]
Guys,when we will be able to Set a custom Url for a Collection ,

From -> https://zeromint.com/collections/w0y98WAUHKbpPslyNBgu8

To -> https://zeromint.com/collections/GoBears

So that other Users cant just Duplicate an NFT( by downloading them ) and Creating a New Collection and Uploading them as an NFT for that Collection with the Same Name

**Comment by r-gochain** [comment]
@treeder good one wdyt?

**Comment by gobullsclub** [comment]
Hello Guys @treeder @r-gochain  ,i notice on [Zeromint](https://www.youtube.com/channel/UC4sTaDc1SMSfYfY3mVt2ubg) Youtube Channel this 
[Sophie Brussaux NFT Art Collection on Zeromint](https://www.youtube.com/watch?v=mBL7GKTu7L4) 
There is a Custom Url [zeromint.com/sophieb](https://zeromint.com/sophieb) ,is this only for Private Listing By Zeromint Team ?

**Comment by treeder** [comment]
Yes, special premium feature. We plan on letting anyone choose a short URL like that down the road.

**Comment by gobullsclub** [comment]
Okay Noted @treeder @r-gochain  ,But how do someone / project apply for this Premium Feature including the Blue Check Mark ?And Whats the Price Like ?
